<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PalGenMixWidget</class>
 <widget class="QWidget" name="PalGenMixWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>169</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="colorWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="redSlider">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>4</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>4</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>159</red>
             <green>160</green>
             <blue>161</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal {background-color: red;}</string>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="greenSlider">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>159</red>
             <green>160</green>
             <blue>161</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal {background-color: lightgreen;}</string>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="blueSlider">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>159</red>
             <green>160</green>
             <blue>161</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::handle:horizontal {background-color: blue;}</string>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="colorEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="toolTip">
          <string>Update sliders from hex value</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="colorPicker">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Drag &amp; Drop</string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/icons/colorpicker.png</normaloff>:/icons/colorpicker.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="GradientWidget" name="gradientWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="colorEditA">
         <property name="toolTip">
          <string>Gradient start color</string>
         </property>
         <property name="text">
          <string>0x000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorPickerA">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Start color picker</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/icons/colorpicker.png</normaloff>:/icons/colorpicker.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="colorCount">
         <property name="toolTip">
          <string>Gradient step count</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="colorEditB">
         <property name="toolTip">
          <string>Gradient end color</string>
         </property>
         <property name="text">
          <string>0xeee</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorPickerB">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip">
          <string>End color picker</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/icons/colorpicker.png</normaloff>:/icons/colorpicker.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="gradientColors">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GradientWidget</class>
   <extends>QWidget</extends>
   <header location="global">gradientwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
